src/components/List.vue

```diff
@@ -56,9 +135,42 @@ export default defineComponent({
   },
   computed: {
     filterex(): Exercise[] {
+      let selectedDifficulty = this.selectedDifficulty;
```

Comment:
use arrow functions down in sort instead, then you dont have to make variables to access `this`